config/locales/interface/slides/en_costs.yml

```diff
@@ -90,7 +90,8 @@ en:
       title: Hydrogen storage
       short_description:
       description: What do you think the costs for hydrogen storage will be in the
-        future?
+        future? The future costs can be adjusted for hydrogen storage in salt caverns and
```

Comment:
As far as I can tell, users can only set the costs of storage **volume** (so in euros/MWh). We have two components for storage costs: volume and capacity. Users currently can't set the capacity component, which is fine. However, I would explain this distinction to the user if possible.